fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/rdf/impl/LdpContainerRdfContext.java

```diff
@@ -203,9 +203,13 @@ public FedoraResourceTripleFunction(final String insertedContainerProperty,
                 return Iterators.transform(values, new Function<Value, Triple>() {
                     @Override
                     public Triple apply(final Value input) {
-                        final RDFNode membershipResource = new ValueConverter(session(), translator())
-                                .convert(input);
-                        return create(subject(), memberRelation, membershipResource.asNode());
+                        try {
+                            final RDFNode membershipResource = new ValueConverter(resource().getNode().getSession(),
```

Comment:
That seems like a good suggestion to me -- I don't see the session getting set anywhere, and doing it in the NodeRdfContext would fix this awkward workaround, and might let us clean up similar workarounds elsewhere.